src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -94,6 +94,14 @@ internal partial class Binder
 
                 foreach (var p in parameterSyntaxList.Value)
                 {
+                    if (p.AttributeLists.Count > 0 && syntax.Kind() == SyntaxKind.AnonymousMethodExpression)
```

Comment:
Gotcha... Can i open an issue on that instead?  I'd prefer to not make any actual changes like that if i can avoid it.  For all i know, attributes *are* allowed on lambdas.  Making a change like that seems like it should go through a new path to get the scrutiny it deserves.